### ISSUE TITLE
New SteamVR button

### ISSUE THREAD

**Issue by somini** [somini/Pixelvision2#43]
There was a beta update that added a SteamVR button.
![Skin](https://cloud.githubusercontent.com/assets/1750665/8970768/71dd2288-3643-11e5-9d20-1e9cd5c16de7.png)
![Default Skin](https://cloud.githubusercontent.com/assets/1750665/8970821/bad67868-3643-11e5-95be-94d63cc7b227.png)
I think the best place to put it would be to the left of the Big Picture mode, just like the default skin.

**Comment by Efreak** [comment]
I wouldn't mind having _both_ buttons moved up there. BPM doesn't need such a big button at the edge of the screen, and without that button there, there's no reason for the URL bar area to be so tall.

Speaking of (re)moving buttons, why is there a dedicated 'add a game' menu at the bottom of the window? It's hardly a frequently-used important function, and all the items in it are in the Games menu.

**Comment by CaptainMagma** [comment]
Sorry, I wasn't aware of this issue earlier. I'll work on a fix for it tomorrow. I think the best thing to do (unless someone is willing to make a custom icon) is to use the default big picture mode button that was added as an override and just add the VR button next to it.

Edit:
By the way, please make sure that you are both opted into the Steam beta.

**Comment by Clavus** [comment]
Don't know if it's related but the downloads page in the library is also broken for me: 

![image](https://cloud.githubusercontent.com/assets/670995/9005094/901c50e6-377b-11e5-9277-3a01c139efa8.png)

**Comment by CaptainMagma** [comment]
For some reason, I am unable to see the Steam VR button even in the default skin. Is there anything I need to do to enable it? Without the ability to see it, I won't be able to fix it.

**Comment by Clavus** [comment]
I believe it only shows up when you have a VR device attached. You would need to disable that behaviour during development I guess, or make a dummy button, position that, and copy the settings over to the VR button.

**Comment by CaptainMagma** [comment]
@Clavus, Have you tested to see what it looks like when you activate Steam VR mode? It appears there is also an "Exit VR" button that needs to be included as well. At least that's what I'm seeing in the code.

**Comment by Clavus** [comment]
Can't test it myself until Monday.  Another thing:  brother said the VR button showed up for him and he doesn't have VR hardware. Are you testing on the steam beta branch?

**Comment by CaptainMagma** [comment]
@Clavus, Yes, I am on the beta branch. Has he ever used VR on his PC? If not, is there anything VR related that he's downloaded?

**Comment by Clavus** [comment]
Maybe if you download SteamVR in the tools tab? He also just moved to Windows 10 but that's probably not related.

**Comment by CaptainMagma** [comment]
@Clavus, Well, I downloaded SteamVR and VRMonitor and tried to run SteamVR but was just brought to some blue setup looking screen. Doubt I'll be able to set anything up without the VR Hardware. I still don't see a button, so I'm kind of at a loss here. If I can't get this to work for me, then someone who has it working will have to fix it. It's a relatively simple fix.

**Comment by Clavus** [comment]
I actually tinkered with it a bit earlier, but somehow my button remained in the top bar instead of next to the big picture button. But anyway, if nobody else can fix it I'll give it another go on Monday.

**Comment by CaptainMagma** [comment]
Thanks @Clavus. If @somini or Mr. Late can't fix it, then I can try to walk you through how to fix it.

**Comment by somini** [comment]
I too can see it without any VR hardware, not sure what's causing it.
Here's the relevant Pressure commit:
https://github.com/DirtDiglett/Pressure-for-Steam/commit/6c11cb073d81801647d9fbe36c448e5730b4c96c

I'll try to have a go at this tomorrow.

**Comment by CaptainMagma** [comment]
@somini, Well all they do there is hide the VR button for some reason instead of adding it. I could do that without being able to see it. They might be hiding it because there's another way they've decided to access VR mode. I know the creator of that skin so I'll just ask him about it tomorrow. I'll let you know what I find out.

**Comment by CaptainMagma** [comment]
I spoke with Dirt Diglett and he told me he hid the VR button so that he could later add it into a drop-down menu since he thinks there are very few people that will make use of the feature. I don't think this is a bad idea either. This way we won't have to remove the custom Big Picture button, so we won't have to adjust the ini file to change the Big Picture button to the default one and create room for the VR button. Let me know what you think of this, @somini.

**Comment by somini** [comment]
Looks good to me. Just add a line to the README to let people know.

**Comment by SHOUBISHOCK** [comment]
![907b91e6-4d74-11e5-965c-2d7f498c87fd](https://cloud.githubusercontent.com/assets/14013250/9543105/60bfc07e-4d76-11e5-931d-e2beb6644ab9.jpg)

text font doesn t match but i wish keep the same spirit
pm me there http://steamcommunity.com/groups/pixelvision2/discussions/0/527273452878509857/

**Comment by Bluscream** [comment]
@SHOUBISHOCK how did you get this?

![image](https://user-images.githubusercontent.com/3318223/63184766-68d5ae00-c058-11e9-90b0-2e0ca1311997.png)

**Comment by Black4Blade** [comment]
> @SHOUBISHOCK how did you get this?
> 
> ![image](https://user-images.githubusercontent.com/3318223/63184766-68d5ae00-c058-11e9-90b0-2e0ca1311997.png)

I at least am not sure what you are referring to
And you are aware that is from the older skin, right? the one that was before the new chat come in?
or is that what you refer to?

**Comment by Bluscream** [comment]
Ohh
